CI/integration_tests/calculators/test_green_kubo_ionic_conductivity.py

```diff
@@ -65,4 +66,12 @@ def test_project(traj_file, true_values, tmp_path):
 
     computation = project.run.GreenKuboIonicConductivity(plot=False)
 
+    # Time is wrong in the test data
```

Comment:
Can you add the correct time to the true values?
Is it possible to write this test without the need for external data?

Reply:
Yep! Before merging I'd like to remove external data here, the time is saved as essentially all zeros... I am looking for an analytical solution

Reply:
To be solved in with #529 associated PR